lua/entities/terminator_nextbot_zambieglasselite/shared.lua

```diff
@@ -9,12 +9,31 @@ list.Set( "NPC", "terminator_nextbot_zambieglasselite", {
     Class = "terminator_nextbot_zambieglasselite",
     Category = "Nextbot Zambies",
 } )
+// TODO: Add spawnicons
+list.Set( "NPC", "terminator_nextbot_zambieglasselite_mega", {
+    Name = "Zombie Glass Elite Mega",
+    Class = "terminator_nextbot_zambieglasselite",
+    Category = "Nextbot Zambies",
+	KeyValues = { iShards = 50 }
+} )
+// This much particles causes the frame to freeze,
+// and I doubt that even setting it to admin only will help!
+//	// This one is purely for comedic effect,
+//	// and has no intention of being, as
+//	// id Software says, "even remotely fair"
+//	list.Set( "NPC", "terminator_nextbot_zambieglasselite_giga", {
+//	    Name = "Zombie Glass Elite Giga",
+//	    Class = "terminator_nextbot_zambieglasselite",
+//	    Category = "Nextbot Zambies",
+//		KeyValues = { iShards = 100 }
+//	} )
 
 if CLIENT then
     language.Add( "terminator_nextbot_zambieglasselite", ENT.PrintName )
 
+	local render_SetColorModulation = render.SetColorModulation
     function ENT:Draw()
-        render.SetColorModulation( 0.5, 0.7, 1.0 )
+        render_SetColorModulation( .5, .7, 1.0 )

```

Comment:
this is an example of GOOD function localisation.
:Draw is ran potentially thousands of times per second ( think 20+ npcs spawned! )